_rules/table-headers-attribute-refer-to-data-cells-a25f45.md

```diff
@@ -0,0 +1,247 @@
+---
+id: a25f45
+name: headers attribute only refers to cells in the same table element.
+rule_type: atomic
+description: |
+  This rule checks that the headers attribute must only refer to cells in the same table element.
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+input_aspects:
+  - DOM Tree
+authors:
+  - Jey Nandakumar
+  - Audrey Maniez
+---
+
+## Applicability
+
+This rule applies to `td` elements with a `headers` attribute, that are [included in the accessibility tree](#included-in-the-accessibility-tree).
+
+## Expectation
+
+Each target element with the `headers` attribute refers to other `th` elements or cells with a [semantic role](#semantic-role) of `columnheader` or `rowheader` within the same `table` element.
```

Comment:
I still think that the text of the Expectation could be improved. What about this:
Each target element refers, in its `headers` attribute, to other `th` elements or cells with a [semantic role](#semantic-role) of `columnheader` or `rowheader` within the same `table` element.